cypress/e2e/Login.cy.js

```diff
@@ -0,0 +1,81 @@
+import { MEMBERS } from "../fixtures/members";
+
+describe("Login behaviour Test", () => {
+  const { UNKNOWN_EMAIL, UNKNOWN_PASSWORD, WRONG_PASSWORD } = MEMBERS;
+
+  beforeEach(() => {
+    // Include `cy.visit()` command in the beforeEach function
+    // so that it runs before each test to visit the same URL at the start of all the tests
+    cy.visit(Cypress.env("REACT_APP_GRAASP_COMPOSE_HOST_STAGE"));
+    // Perform an assertion with `should` to verify redirection
+    cy.url().should(
+      "be.equal",
+      Cypress.env("REACT_APP_AUTHENTICATION_HOST_STAGE") + "/signin"
```

Comment:
It could be interesting to map the env variables to more common names (`REACT_APP_AUTHENTICATION_HOST`) if we want one day to reuse those tests for dev.

Reply:
I updated the variables to more common names for better understanding. However, I don't think we should reuse these tests for dev anytime as that is not their purpose. Integration testing should always be performed inside the stage environment